src/msg.rs

```diff
@@ -43,6 +43,15 @@ pub struct RenderQueryMsg {
     pub context: Option<Value>,
 }
 
+#[cw_serde]
+pub struct NameRecordsQueryMsg {
+    pub limit: u8,
+    pub cursor: Option<String>,
+    pub network_den: Option<String>
```

Comment:
Often times, what you call `network_den` here is referred to as `prefix` in cosmwasm contexts. Could possibly rename it to that (or more fully `network_prefix`)

Reply:
done